packages/dependency-extraction-webpack-plugin/src/index.js

```diff
@@ -17,14 +17,18 @@ function camelCaseDash( string ) {
 	return string.replace( /-([a-z])/g, ( _, letter ) => letter.toUpperCase() );
```

Comment:
These changes allowed me to more easily exclude a package if needed by doing this:
```
new WooCommerceDependencyExtractionWebpackPlugin( {
	excludedExternals: [ 'components' ],
} ),
```